java/ql/src/semmle/code/java/frameworks/guava/IO.qll

```diff
@@ -60,7 +62,20 @@ private class GuavaIoCsv extends SummaryModelCsv {
         "com.google.common.io;MoreFiles;false;getFileExtension;(Path);;Argument[0];ReturnValue;taint",
         "com.google.common.io;MoreFiles;false;getNameWithoutExtension;(Path);;Argument[0];ReturnValue;taint",
         "com.google.common.io;LineReader;false;LineReader;(Readable);;Argument[0];ReturnValue;taint",
-        "com.google.common.io;LineReader;true;readLine;();;Argument[-1];ReturnValue;taint"
+        "com.google.common.io;LineReader;true;readLine;();;Argument[-1];ReturnValue;taint",
+        "com.google.common.io;ByteArrayDataOutput;true;toByteArray;();;Argument[-1];ReturnValue;taint",
+        "com.google.common.io;ByteArrayDataOutput;true;write;(byte[]);;Argument[0];Argument[-1];taint",
+        "com.google.common.io;ByteArrayDataOutput;true;write;(byte[],int,int);;Argument[0];Argument[-1];taint",
+        "com.google.common.io;ByteArrayDataOutput;true;writeByte;(int);;Argument[0];Argument[-1];taint",
```

Comment:
If `writeByte` is covered, it might be good to cover [`write(int)`](https://javadoc.io/static/com.google.guava/guava/30.1-jre/com/google/common/io/ByteArrayDataOutput.html#write(int)) as well.